README.md

```diff
@@ -9,3 +9,36 @@ To stop this from happening, this bot shares a ban list of known scammers to any
 This bot is powered via community reporting, trusted verification, and complete transparency in all actions.
 
 You can read more on [the website here](https://theantiscamgroup.com)!
+
+## Configurations to Run the Bot
+
```

Comment:
I think it might be good to mention that setting the configs is only necessary if you're going to run your own instance of the bot. Just so people don't get confused and think they have to do this to install the bot in the TAG server.